test/smoke/twilioenablebysms_test.go

```diff
@@ -12,9 +12,9 @@ func TestTwilioEnableBySMS(t *testing.T) {
 	t.Parallel()
 
 	sqlQuery := `
-		insert into users (id, name, email) 
+		insert into users (id, name, email, role) 
 		values 
-			({{uuid "user"}}, 'bob', 'joe');
+			({{uuid "user"}}, 'bob', 'joe', 'admin');
```

Comment:
Prior these tests were using the harness' GraphQLQuery2 method which internally uses a admin user to make the requests. Because we added a check where only the contact methods owner can send a test these would all fail as the user that is inserted into the database != the admin user the GraphQLQuery2 method was using.

That all being said it seems at least the user role would be required, I did the admin one to keep it similar to how the test was previously running with a admin user being used.

I'll leave this one unresolved in the event there is another way we want to handle this?

Reply:
Got it, yeah default role is `unknown`, which would have an issue. Thanks for clarifying.

That said, I'd change the role to `user` to err on the side of caution; that way, the tests validate with the least permission instead of the most. If we were ever in a state where only admins could validate contact methods, we'd want this test to fail, for example.

Reply:
Update the role to `user`.